test/best_listing_mixin/best_listing_mixin_test.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2018, the Dart Reddit API Wrapper project authors.
```

Comment:
General comment: when adding new files, the copyright year should be the current year.